### ISSUE TITLE
translation into Spanish

### ISSUE THREAD

**Issue by Dunhill69** [franciscoBSalgueiro/en-croissant#413]
### Describe the feature

I would like to contribute with the translation into Spanish. Could you please send me the language file so I can translate it?
Thank you very much

**Comment by franciscoBSalgueiro** [comment]
It's this one src/translation/es_ES.ts

**Comment by Dunhill69** [comment]
Yes, but the translation is not complete. There are words that, even if you
choose Spanish, the interface maintains words in the original language.
